.github/workflows/release.yml

```diff
@@ -232,35 +232,26 @@ jobs:
           fi
           git checkout $BRANCH_NAME
 
-          # update veneer file
-          yq -i '.Fast.Bundles += [{"Image": "docker.io/perconalab/everest-operator-bundle:" + strenv(VERSION) }]' catalog/everest-operator/veneer.yaml
-          if [[ ! $env.IS_RC ]]; then
-            yq -i '.Stable.Bundles += [{"Image": "docker.io/percona/everest-operator-bundle:" + strenv(VERSION) }]' catalog/everest-operator/veneer.yaml
+          # if there is something to commit, commit it and add the tag
+          if git tag --list | grep -q "^$GH_TAG$"; then
+            echo "The tag is already present in github. Please create a different RC/Release"
+            exit 1
           fi
 
-          curl -Lo /tmp/opm https://github.com/operator-framework/operator-registry/releases/latest/download/${OS}-${ARCH}-opm
-          chmod +x /tmp/opm
-          /tmp/opm alpha render-template semver -o yaml < ./catalog/everest-operator/veneer.yaml > ./catalog/everest-operator/catalog.yaml
+          # configure userdata for commits
+          git config --global user.email "everest-ci@percona.com"
+          git config --global user.name "Everest RC CI triggered by ${{ github.actor }}"
 
-          # if there is something to commit, commit it and add the tag
-          if [[ -n $(git status --porcelain) ]]; then
-            if git tag --list | grep -q "^$GH_TAG$"; then
-              echo "The tag is already present in github. Please create a different RC/Release"
-              exit 1
-            fi
-
-            # configure userdata for commits
-            git config --global user.email "everest-ci@percona.com"
-            git config --global user.name "Everest RC CI triggered by ${{ github.actor }}"
-
-            # commit and push the updated files
-            git commit -a -m "update version tag"
-            git push origin $BRANCH_NAME
-          
-            git tag $GH_TAG
-            git push origin $GH_TAG
+          # Check if veneer has the new version listed
+          if grep -q "$VERSION" catalog/everest-operator/catalog.yaml; then
+            echo "catalog/everest-operator/catalog.yaml does not include the version $VERSION"
+            echo "Update the file manually before running the workflow"
+            exit 1
```

Comment:
I'm thinking if can actually update it before running this workflow :thinking: 
In order to generate the new catalog we need the operator bundle image to be available in the public registry and that image is only built and pushed as part of this workflow.

Reply:
@oksana-grishchenko what I just said is true?

Reply:
As discussed offline:

> the process would be:
> 1. run the release pipeline with 0.10.0-rc3 input
> 2. wait until it exits on the catalog step
> 3. go to the catalog release branch, edit the catalog manually, run the generation manually, commit the changes to the release branch
> 4. run the release pipeline again with 0.10.0-rc3 input
> the pipeline does for the operator in case we run the same tag again - it shouldn’t fail. If there is no changes the only thing it will do is to push a new build to the dockerhub (to the same tags), which is not ideal but we can afford it. And the Michal’s branch already contains the exit logic for the catalog if there is no mention of the version.
> So the process I described above should work just fine.

Reply:
This process should work fine with this PR:

1. run the release pipeline with `0.10.0-rc3` input
2. wait until it exits on the catalog step
3. go to the catalog release branch, edit the catalog manually, run the generation manually, commit the changes to the release branch
4. run the release pipeline again with `0.10.0-rc3` input